Lib/test/eintrdata/eintr_tester.py

```diff
@@ -516,7 +516,7 @@ def _lock(self, lock_func, lock_name):
 
     # Issue 35633: See https://bugs.python.org/issue35633#msg333662
     # skip test rather than accept PermissionError from all platforms
-    @unittest.skipIf(platform.system() == "AIX", "AIX returns PermissionError")
+    @unittest.skipIf(platform.system() in ("AIX", "OS400"), "AIX and OS400 returns PermissionError")
```

Comment:
Did this not get fixed with the lockf change?

Reply:
should be fixed here as well, I forgot to remove skips here